source/view/wizard/pages/equations_page.py

```diff
@@ -24,7 +27,8 @@ def __init__(self):
 
     def set_content_attributes(self):
         self.equation.setPlaceholderText("insert your equation here...")
-        self.equation.setFont(QFont("Cambria Math", 12))
+        self.equation.setFont(QFont("DejaVu Sans", 12))
```

Comment:
```suggestion
        self.equation.setFont(QFont("Cambria Math", 12))
```

Needs to be changed back to Cambria Math